develop/blockchains/maintenance/runtime-metrics-monitoring.md

```diff
@@ -0,0 +1,107 @@
+
+As a developer or node operator, the telemetry system handles most of the technical setup. Collected data is automatically sent to a default telemetry server, where it’s aggregated and displayed on a dashboard, making it easy to monitor network performance and identify issues.
+
+## Runtime Metrics
+
+Substrate exposes a variety of metrics about the operation of your network, such as the number of peer connections, memory usage, and block production. To capture and visualize these metrics, you can configure and use tools like [Prometheus](https://prometheus.io/){target=\_blank} and [Grafana](https://grafana.com/){target=\_blank}. At a high level, Substrate exposes telemetry data that can be consumed by the Prometheus endpoint and then presented as visual information in a Grafana dashboard or graph. The provided diagram offers a simplified overview of how the interaction between Substrate, Prometheus, and Grafana can be configured to display information about node operations.
+
+```mermaid
+graph TD
+  subnode([Substrate node]) --> telemetrystream[Exposed telemetry stream]
+  telemetrystream --> prom[Prometheus]
+  configuration[Configuration: Every 1 minute] --> prom
+  prom --> endpoint[Endpoint: Every 1 minute]
+  endpoint --> grafana[Grafana]
+  grafana --> useropen[User opens a graph]
+  prom --> localdata[Local Prometheus data]
+  localdata --> getmetric
+```
+
+The diagram shows the flow of data from the Substrate node to the monitoring and visualization components. The Substrate node exposes a telemetry stream, which is consumed by Prometheus. Prometheus is configured to collect data every minute and store it. Grafana is then used to visualize the data, allowing the user to open graphs and retrieve specifc metrics from the telemetry stream.
+
+## Visual Monitoring
+
+The [Polkadot telemetry](https://telemetry.polkadot.io/){target=\_blank} dashboard provides a real-time view of how currently online nodes are performing. This dashboard, allows users to select the network you need to check on, and also the information you want to display by turning visible columns on and off from the list of columns available. The monitoring dashboard provides the following indicators and metrics:
+
+- **Validator** - identifies whether the node is a validator node or not
+- **Location** - displays the geographical location of the node
+- **Implementation** - shows the version of the software running on the node
+- **Network ID** - displays the public network identifier for the node
+- **Peer count** - indicates the number of peers connected to the node
+- **Transactions in queue** - shows the number of transactions waiting in the [Ready queue](https://paritytech.github.io/polkadot-sdk/master/sc_transaction_pool_api/enum.TransactionStatus.html#variant.Ready){target=\_blank} for a block author
```

Comment:
```suggestion
- **Transactions in queue** - shows the number of transactions waiting in the [`Ready` queue](https://paritytech.github.io/polkadot-sdk/master/sc_transaction_pool_api/enum.TransactionStatus.html#variant.Ready){target=\_blank} for a block author
```

This can either be "`Ready` queue" as it's a variant, or casually referred to as the "ready queue"